docs/sphinx/dev_guide/contributing.rst

```diff
@@ -153,24 +174,26 @@ be run on PRs that are made on branches within the GitHub RAJA repository.
 
           If successful, this will create a branch in your local copy of the
           RAJA repo labeled ``pr-from-fork/<PR #> and you will be on that
```

Comment:
Is there a closing `` missing from here? It doesn't seem to mess up the file when I view it, but it looks like it's missing the ``...

Reply:
Lol I don't know how to type them without git thinking I want to type code...

Reply:
Yup. Got it. Thanks.